pkgs/applications/version-management/gitlab/gitaly/default.nix

```diff
@@ -20,6 +20,18 @@ let
         };
       };
   };
+
```

Comment:
If this override is likely do be needed by other packages (due to the git2go dependency?) it's better to define another version in in `top-level/all-packages.nix`

Reply:
I agree, we should expose it

Reply:
done :heavy_check_mark:

Reply:
That thread would have answered my question if it wasn't marked as resolved. GitHub... 😒